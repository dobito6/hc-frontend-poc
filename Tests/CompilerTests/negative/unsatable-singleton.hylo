fun check<T>(_ x: T) {}

fun g(x: sink Void) {
  let y: #exactly(x)
}
