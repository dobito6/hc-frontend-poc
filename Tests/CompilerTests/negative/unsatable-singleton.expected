negative/unsatable-singleton.hylo:4.19-20: error: expression is not stable
  let y: #exactly(x)
                  ^
